src/physics/operators/odes.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <functional>
+#include <variant>
+
+#include "mfem.hpp"
+#include "numerics/expr_template_ops.hpp"
+#include "physics/utilities/boundary_condition_manager.hpp"
+#include "physics/utilities/equation_solver.hpp"
+
+class SecondOrderODE : public mfem::SecondOrderTimeDependentOperator {
+public:
+  using signature = void(const double, const double, const double, const mfem::Vector&, const mfem::Vector&,
```

Comment:
I'll change it to "TypeSignature" to be more in agreement with the cs terminology. I would prefer to not append ODE to the prefix, since it would make the full name redundant
`SecondOrderODE::ODESignature`

I didn't really think about it until now, but I guess `using` statements follow the same naming conventions of other types, so I'll update them accordingly to CamelCase.